docs/computing/webinterface/index.md

```diff
@@ -63,30 +68,32 @@ The file browser comes with a basic text editor. Some important notes on that:
 - There is no _save-as_ feature
 - If a read-only file is opened no indication will be given to the user but no changes will be applied
 
+
 ### Active jobs
 
 Recent and running batch jobs can be viewed using the _Jobs_ section on the top navbar and selecting _Active jobs_. Here you can view the current status of the job and what kind of resources were requested. Deleting a running job will cancel the job. 
 
 In the future it will become possible to submit batch jobs through the web interface, but for now the recommended way is to launch the jobs using `sbatch` from the shell.   
 
+
 ### Interactive apps
 
 Interactive apps are programs that can be launched and run on the compute nodes and provide a web interface.
 These are apps such as Jupyter Notebook, RStudio, Visual Studio Code and Rclone. 
```

Comment:
if we have complete list in beginning, this does not need to be complete at least. In any case rclone should be removed, it is not available.